src/styles/globals.css

```diff
@@ -184,7 +183,3 @@
     justify-content: center;
   }
 }
-
-.dark {
```

Comment:
this normally doesn't apply a class, can we use a modifier for the background colour instead?